### PR TITLE
DOCSP-37059: Fix build errors

### PR DESCRIPTION
**Pull request by norareidy**
JIRA - https://jira.mongodb.org/browse/DOCSP-37059
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/mongoid/DOCSP-37059/
Build log - https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65d8c9c67111165982a4d7af

### CODE REVIEW

File: docs/release-notes/mongoid-9.0.txt [hunk 1]
```diff
@@ -522,14 +522,14 @@ to ``nil`` values. If you are using Rails, the default time zone is already
 set to UTC. If you are not using Rails, you may set a time zone at the start
 of your program like this:
 
-... code-block:: ruby
+.. code-block:: ruby
 
   Time.zone = 'UTC'
```
Comment (jordan-smith721) [inline docs/release-notes/mongoid-9.0.txt:527]:
```suggestion
   Time.zone = 'UTC'
```

File: docs/release-notes/mongoid-9.0.txt [hunk 2]
```diff
@@ -522,14 +522,14 @@ to ``nil`` values. If you are using Rails, the default time zone is already
...
 
 This will set the time zone to UTC. You can see all available time zone names
 by running the following command:
 
-... code-block:: bash
+.. code-block:: bash
 
   $ ruby -ractive_support/values/time_zone \
     -e 'puts ActiveSupport::TimeZone::MAPPING.keys'
```
Comment (jordan-smith721) [inline docs/release-notes/mongoid-9.0.txt:535]:
```suggestion
   $ ruby -ractive_support/values/time_zone \
     -e 'puts ActiveSupport::TimeZone::MAPPING.keys'
```

File: docs/release-notes/mongoid-9.0.txt [hunk 3]
```diff
@@ -560,7 +560,7 @@ a different database, or a different collection).
 If you need the legacy (pre-9.0) behavior, you can enable it with the following
 flag:
 
-... code-block:: ruby
+.. code-block:: ruby
 
   Mongoid.legacy_persistence_context_behavior = true
```
Comment (jordan-smith721) [inline docs/release-notes/mongoid-9.0.txt:565]:
```suggestion
   Mongoid.legacy_persistence_context_behavior = true
```

File: docs/tutorials/automatic-encryption.txt [hunk 4]
```diff
@@ -64,9 +64,11 @@ This can be done one of two ways.
 Install the automatic encryption shared library (Ruby driver 2.19+)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you use the Ruby driver version 2.19 and above the automatic encryption
-shared library should be installed by following the instructions in the
-`MongoDB manual <https://www.mongodb.com/docs/manual/core/queryable-encryption/reference/shared-library/#download-the-automatic-encryption-shared-library>`_.
+If you use the Ruby driver version 2.19 and above, the automatic encryption
+shared library should be installed by following the instructions on the
+`Automatic Encryption Shared Library for Queryable Encryption 
+<https://www.mongodb.com/docs/manual/core/queryable-encryption/reference/shared-library/#download-the-automatic-encryption-shared-library>`_
```
Comment (jordan-smith721) [inline docs/tutorials/automatic-encryption.txt:70]: We have a tag for pages in the Server manual that is a bit shorter

```suggestion
:manual:`Automatic Encryption Shared Library for Queryable Encryption 
</core/queryable-encryption/reference/shared-library/#download-the-automatic-encryption-shared-library>`
```

File: docs/tutorials/automatic-encryption.txt [hunk 5]
```diff
@@ -253,9 +255,9 @@ Now we can tell Mongoid what should be encrypted:
 Known Limitations
 ~~~~~~~~~~~~~~~~~
 
-* MongoDB CSFLE has some limitations that are described in
-  `the server documentation. <https://www.mongodb.com/docs/manual/core/csfle/reference/limitations/>`_
-  These limitations also apply to Mongoid.
+* MongoDB CSFLE has some limitations that are described on the
+  `CSFLE Limitations <https://www.mongodb.com/docs/manual/core/csfle/reference/limitations/>`_
+  page in the Server manual. These limitations also apply to Mongoid.
```
Comment (jordan-smith721) [inline docs/tutorials/automatic-encryption.txt:260]:
```suggestion
  :manual:`CSFLE Limitations </core/csfle/reference/limitations/>`
  page in the Server manual. These limitations also apply to Mongoid.
```

**Review by jordan-smith721 — CHANGES_REQUESTED** [review]
Just a few small changes

**Review by alexbevi — APPROVED** [review]
LGTM

**Pull request merged**